### PR TITLE
<MarketingLayout/> - add size tiny

### PR DESCRIPTION
**Pull request by yogevitz**
### 🔦 Summary

add `size: tiny` for `<MarketingLayout/>` component

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers @EgoziE 
- [x] 👨‍🎨 UX change is approved by the Design System UX @milkyfruit 
- 📚 Change is documented
  - [x] Story
  - [x] API description
- 🔬 Change is tested
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6269

File: src/MarketingLayout/components/Title.js [hunk 1]
```diff
@@ -3,14 +3,19 @@ import Heading from '../../Heading';
...
 import PropTypes from 'prop-types';
 
 const Title = ({ children, appearance }) => (
   <div className={classes.title}>
     {isString(children) ? (
-      <Heading dataHook={MARKETING_LAYOUT_TITLE} appearance={appearance}>
-        {children}
-      </Heading>
+      appearance === SIZES.tiny ? (
+        <div>{children}</div>
```
Comment (EgoziE) [inline src/MarketingLayout/components/Title.js:13]: Missing data-hook

```suggestion
        <div data-hook={MARKETING_LAYOUT_TITLE}>{children}</div>
```

**Review by EgoziE — COMMENTED** [review]
@milkyfruit Ready for UX review

**Review by milkyfruit — CHANGES_REQUESTED** [review]
@yogevitz @EgoziE 
- [x] For size tiny **text column span** should be `8` instead of `9`
- [ ] For size tiny **image column span** should be `3` instead of `2`
- [ ] Let's slightly adjust example to show image size as in spec:
```
<MarketingLayout
  title="Tiny Size Layout"
  description="This layout requires less attention. It can promote side features that might add value, but are not mandatory to achieve main goals."
  actions={<Button size="small">Get Started</Button>}
  size="tiny"
  image={
    <Box width="100%" align="right">
      <Image width="96px" height="96px" />
    </Box>
  }
/>;
```

Preview in the spec:
image

Everything else looks great, after layout is fixed, we can merge it 👍

**Comment by yogevitz** [comment]
@milkyfruit column spans and example fixed 👍

**Review by milkyfruit — APPROVED** [review]
@yogevitz @EgoziE looks great!

**Review by EgoziE — APPROVED** [review]

**Pull request merged**